tuples_test.go

```diff
@@ -391,45 +392,58 @@ func Test_DeleteRelationshipTuple(t *testing.T) {
 	fc := NewMockFGAClient(t, c)
 
 	testCases := []struct {
-		name        string
-		relation    string
-		object      string
-		expectedRes string
-		errRes      string
+		name              string
+		relation          string
+		object            string
+		expectedRes       string
+		errRes            string
+		numTuplesToDelete int
 	}{
 		{
-			name:        "happy path with relation",
-			object:      "organization:datum",
-			relation:    "member",
-			expectedRes: "",
-			errRes:      "",
+			name:              "happy path with relation",
+			object:            "organization:datum",
+			relation:          "member",
+			expectedRes:       "",
+			numTuplesToDelete: 12,
+			errRes:            "",
 		},
 		{
-			name:        "error, missing relation",
-			object:      "organization:datum",
-			relation:    "",
-			expectedRes: "",
-			errRes:      "Reason: the 'relation' field is malformed",
+			name:              "error, missing relation",
+			object:            "organization:datum",
+			relation:          "",
+			expectedRes:       "",
+			numTuplesToDelete: 30,
```

Comment:
This isn't being tested here so I would leave the error ones at 1
```suggestion
			numTuplesToDelete: 1,
```